File: src/gestorAplicacion/Hija/MensajeSocial.java
package gestorAplicacion.Hija;

import gestorAplicacion.Padre.*;

//Son mensajes informales
class MensajeSocial extends Mensaje{
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String nombreA;
	private String nombreB;
	
	//Crea mensaje con tilulo y cuerpo
	MensajeSocial(String titulo, String cuerpo, String nombreA, String nombreB) {
		super(titulo, cuerpo);
		this.nombreA = nombreA;
		this.nombreB = nombreB;
	}
	
	@Override
	public String toString() {
		return "MensajeSocial [titulo=" + this.getTitulo() + ", cuerpo=" + this.getCuerpo() 
		+ ", nombreA=" + getNombreA() + ", nombreB=" + getNombreB() + "]";
	}

	String getNombreA() {
		return nombreA;
	}

	String getNombreB() {
		return nombreB;
	}

}

File: Python/uiMain/gestorAplicacion/Hija/MensajeSocial.py
from gestorAplicacion.Padre.Mensaje import Mensaje
class MensajeSocial(Mensaje):

    def __init__(self, titulo: str, cuerpo: str, nombreA: str, nombreB: str):
        super().__init__(titulo, cuerpo)
        self._nombreA = nombreA
        self._nombreB = nombreB

    def getNombreA(self):
        return self._nombreA

    def getNombreB(self):
        return self._nombreB

    def __str__(self) -> str:
        return "MensajeSocial [titulo=" + self.getTitulo() + ", cuerpo=" + self.getCuerpo() + ", nombreA=" + self.getNombreA() + ", nombreB=" + self.getNombreB() + "]"